docs/communicate-with-javascript.md

```diff
@@ -1267,6 +1332,9 @@ compile the first argument of the JavaScript function to the string `"exit"`.
 You can also use any JSON literal by passing a quoted string to `bs.as`: `bs.as
 {json|true|json}` or `bs.as {json|{"name": "John"}|json}`.
 
+Sometimes, passing a constant argument to an external function is not enough.
```

Comment:
"is not enough" for what? I am not sure I get the connection between this "constant values as arguments" section and the `bs.inline` attribute 🤔

Reply:
this connection probably doesn't make sense indeed. I'll remove.